.travis.yml

```diff
@@ -3,20 +3,23 @@ language: php
 
 # list any PHP version you want to test against
 php:
-  - 7.0
-  - 7.1
-  - 7.2
+  - '7.0'
+  - '7.1'
+  - '7.2'
+env:
+  - COMPOSER_PARAMETER=--prefer-lowest
+  - COMPOSER_PARAMETER=
 
 # optionally set up exclutions and allowed failures in the matrix
 matrix:
   fast_finish: true
 
 install:
   - travis_retry composer config -g github-oauth.github.com $GH_TOKEN
-  - travis_retry composer install
+  - travis_retry composer update $COMPOSER_PARAMETER
```

Comment:
jetzt doch update?

Reply:
Ja. Der Unterschied ist, dass es hier in dem Repo absichtlich keine composer.lock gibt (weil halt eine Lib). Das update benutze ich um in einem Fall die kleinst möglichsten Dependencies zu installieren und im anderen Fall die neuesten. Damit ist sicher gestellt, dass wir den Coding-Standard auch in Zukunft auf mehreren Projekten benutzen können.